machine1/dbus.go

```diff
@@ -191,3 +202,59 @@ func (c *Conn) ListMachines() ([]MachineStatus, error) {
 
 	return machs, nil
 }
+
+func imageFromInterfaces(image []interface{}) (*ImageStatus, error) {
+	if len(image) < 6 {
```

Comment:
I think this should be `< 7`, right? Otherwise we may be off by one on the last `jobpath` (if I am not miscounting).